src/js/components/CartManger.js

```diff
@@ -0,0 +1,102 @@
+import App from "../App";
+export class CartManger {
+  /**
+   * @type {Cart[]} #productId
+   */
+  #cartList;
+
+  constructor() {
+    this.#cartList = [];
+  }
+
+  /**
+   * Push Cart to #cartList or increase the number of #quantity
+   * @param {Cart} cart
+   */
+  addCart(cart) {
+    const isCartItemInCart = this.#cartList.some((cartItem) => {
+      return cartItem.getProductId() == cart.getProductId();
+    });
+
+    if (!isCartItemInCart) {
+      this.#cartList.push(cart);
+    } else {
+      cart.increaseQuantity();
+    }
+  }
```

Comment:
Just like the `removeCart()`, could you make it pass `productId` to add the product to the cart.
Therefore, could you instantiate a `Cart` inside this method and push it to the list, not pushing the received `Cart` to it?